docs/compatibility.md

```diff
@@ -17,5 +17,14 @@ but also provide lower-cased versions for API compatibility with `requests`.
 
 ## Advanced Usage
 
-!!! warning
-    TODO
+### requests.Session
+The HTTPX equivalent of `requests.Session` is `httpx.Client`.
+
+```python
+session = requests.Session(**kwargs)
+```
+is equivalent to
+```python
+client = httpx.Client(**kwargs)
+```
+More detailed documentation and usage of `Client` can be found in [Advanced Usage](advanced.md).
```

Comment:
Sorry I didn't get this in my first review — should we try to keep the Markdown formatting consistent here? I believe it is encouraged to put lines between various elements, i.e.:

```markdown
### requests.Session

The HTTPX equivalent of `requests.Session` is `httpx.Client`.

\```python
session = requests.Session(**kwargs)
\```

is equivalent to:

\```python
client = httpx.Client(**kwargs)
\```

More detailed documentation and usage of `Client` can be found in [Advanced Usage](advanced.md).
```